worked-solutions/01_penguin_classification_solutions.ipynb

```diff
@@ -694,49 +699,29 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Epoch 0 time: 0.146 seconds.\n",
-      "Epoch 1 time: 0.096 seconds.\n",
-      "Epoch 2 time: 0.093 seconds.\n",
-      "Epoch 3 time: 0.092 seconds.\n",
-      "Epoch 4 time: 0.093 seconds.\n",
-      "Epoch 5 time: 0.093 seconds.\n",
-      "Epoch 6 time: 0.092 seconds.\n",
-      "Epoch 7 time: 0.093 seconds.\n",
-      "Epoch 8 time: 0.093 seconds.\n",
-      "Epoch 9 time: 0.094 seconds.\n",
-      "Epoch 10 time: 0.097 seconds.\n",
-      "Epoch 11 time: 0.097 seconds.\n",
-      "Epoch 12 time: 0.094 seconds.\n",
-      "Epoch 13 time: 0.094 seconds.\n",
-      "Epoch 14 time: 0.093 seconds.\n",
-      "Epoch 15 time: 0.092 seconds.\n",
-      "Epoch 16 time: 0.096 seconds.\n",
-      "Epoch 17 time: 0.098 seconds.\n",
-      "Epoch 18 time: 0.096 seconds.\n",
-      "Epoch 19 time: 0.093 seconds.\n",
+      "Epoch 0-25 time: 4.040087 seconds\n",
+      "Epoch 25-50 time: 4.014584 seconds\n",
+      "Epoch 50-75 time: 4.016838 seconds\n",
+      "Epoch 75-100 time: 3.246220 seconds\n",
+      "Epoch 100-125 time: 2.165241 seconds\n",
+      "Epoch 125-150 time: 3.849997 seconds\n",
+      "Epoch 150-175 time: 3.943952 seconds\n",
+      "\n",
```

Comment:
This seems to take longer to train now, why?
Again, asking for my own interest rather than as a criticism.

Reply:
Probably because the learning rate dropped. We could revert this change.